pyiron_base/jobs/flex/executablecontainer.py

```diff
@@ -43,6 +49,7 @@ def __init__(self, project, job_name):
         super().__init__(project, job_name)
         self._write_input_funct = None
         self._collect_output_funct = None
+        self._python_only_job = True
```

Comment:
`ExecutableContainerJob` is also used for `wrap_executable` on the project level, right?  Is this flag then necessary force certain behavior?  Would be nice to mention this here in a comment.

Reply:
The general motivation is to replace the previous `run_static()` functionality which calls `write_input()`, the executable and afterwards `collect_output()` with a more functional approach which generates a `calculate()` function with the corresponding arguments using `generate_calculate_function()` and `generate_calculate_function_kwargs()`. This calculate function is then called, at the moment in the same Python process but in future this can be submitted using via an `concurrent.futures.Executor`. After the successful execution the output is stored using the `job._store_output()` function. 

At the moment we use the existing `self._python_only_job` flag to only call `run_static()` rather than the full `run_static()` stack.

Reply:
> At the moment we use the existing `self._python_only_job` flag to only call `run_static()` rather than the full `run_static()` stack.

A-ok with me, but this is not clear from the code, so it'd be nice to have a comment there.

Reply:
done.